client/src/plugins/process-applications/ProcessApplicationsPlugin.js

```diff
@@ -91,8 +92,35 @@ export default function ProcessApplicationsPlugin(props) {
         setProcessApplicationItems([]);
       }
     });
+
+    subscribe('bpmn.modeler.configure', ({ middlewares, tab }) => {
+
+      if (tab.type !== 'cloud-bpmn') {
+        return;
+      }
+
+      middlewares.push(config => {
+        return {
+          ...config,
+          additionalModules: [
+            ...config.additionalModules || [],
+            {
+              processApplications: [ 'value', processApplications ]
```

Comment:
I'm still not a fan of this as `processApplications` is not a bpmn-js extension and for clarity I'd like to maintain a clear separation between bpmn-js and the surrounding application. We might want to refactor this later.

Reply:
What is your suggestion?

Reply:
I'll edit this so that ResourcesProvider gets only what it needs. If you have another idea, let's reconsider.

Reply:
Now on the branch.